### DISCUSSION TITLE
Outside collaborators on smol-rs/smol

### DISCUSSION THREAD

**Discussion by notgull** [smol-rs/smol#286] [General]
Hello @zkat, @dignifiedquire, @withoutboats, @spacejam, @sdroege, @serzhiio, @llebout and @k-nasa.

I noticed that all of you have write access to the `smol-rs/smol` repo, but no access to any of the other repositories in `smol-rs`. It seems that this write access was added back when `smol` was just a single Git repo under Stjepan's GitHub account, before all of the repos were transferred to this organization.

Do any of you still have interest in contributing to `smol`? If so, I can add you as members of `smol-rs` so you can have access to all of the repositories. I could always use an extra hand for reviewing patches and maintenance. If not, do you mind if I revoke your write access from the `smol` repo, just to keep the `smol-rs` permissions clean?

**Comment by zkat** [comment]
Please go ahead and remove me. My access is from the early days before this was public, I believe

**Reply by sdroege** [reply]
Same for me

**Reply by k-nasa** [reply]
Same for me

**Comment by withoutboats** [comment]
(NOT A CONTRIBUTION) Please remove my access

**Comment by dignifiedquire** [comment]
Still looking at issues here and there, but nothing enough to warrant full access.. So you can remove me for now. If I find the time to contribute more I'll do that first before requesting access :)
